tasks/install.yml

```diff
@@ -74,3 +72,5 @@
     dest: /etc/td-agent/plugin
   with_items: "{{ fluentd_plugins }}"
   notify: restart td-agent
+
+- meta: flush_handlers
```

Comment:
Why???

Reply:
Well, in fact I've just left it as it was before. I'm not sure if it is essential - I will check it tomorrow